other/materials_designer/create_heterostructure_example.ipynb

```diff
@@ -0,0 +1,765 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    MAX_AREA_01 = 50 # search are for the first interface
</pre><p>area</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>